nevergrad/optimization/optimizerlib.py

```diff
@@ -386,6 +390,7 @@ def __init__(
 DoubleFastGADiscreteOnePlusOne = ParametrizedOnePlusOne(mutation="doublefastga").set_name(
     "DoubleFastGADiscreteOnePlusOne", register=True
 )
+RLSOnePlusOne = ParametrizedOnePlusOne(mutation="rls").set_name("RLSOnePlusOne", register=True)
```

Comment:
as usual, is that something this is important to the point of being added by default? It's actually better documented by ParametrizedOnePlusOne, there is no way to document all the optimizers added here.